README.md

```diff
@@ -16,7 +16,7 @@
 ## Requirements
 
 - `python >=3.8`
-- `jupyterlite >=0.1.0b19`
+- `jupyterlite-core >=0.2.0a0`
```

Comment:
Technically end users can still stick to JupyterLite 0.1.0 for now.

Or maybe the README should mention `jupyterlite-core >=0.2.0a0` for `jupyterlite-pyodide-kernel >=0.2.0a0`.

Reply:
Yep, need to start the compatibility table (bleah).

Reply:
09b9e75